### DISCUSSION TITLE
Unknown error message when validating secrets

### DISCUSSION THREAD

**Discussion by Juli0787** [community/community#106924] [New to GitHub]
When validating my secrets I received the following error message 'Process completed with exit code 2' and I cannot find a solution in the according docs. Anyone who can help?

**Comment by queenofcorgis** [comment]
Hi @Juli0787 ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same content in multiple places. As a result, we have closed this duplicate discussion, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).

**Comment by Juli0787** [comment]
Hej Lili,

Thanks for your note. It wasn't on purpose at all, think I was or am just confused where to post my error. Can I still come back to you with the full code and which discussion forum would be best?

Thanks,
Julia

Gesendet von Outlook für iOS<https://aka.ms/o0ukef>
________________________________
Von: Lili ***@***.***>
Gesendet: Tuesday, February 13, 2024 3:27:34 PM
An: community/community ***@***.***>
Cc: Juli0787 ***@***.***>; Mention ***@***.***>
Betreff: Re: [community/community] Unknown error message when validating secrets (Discussion #106924)


Hi @Juli0787<https://github.com/Juli0787> ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same content in multiple places. As a result, we have closed this duplicate discussion, and we ask that you no longer cross-post.

For additional guidance, please review our Community on Discussions Code of Conduct<https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md> and GitHub’s Terms of Service<https://docs.github.com/en/github/site-policy/github-terms-of-service>.

**Reply by queenofcorgis** [reply]
No worries at all - we're happy to have you. Please do share your code in the [open discussion](https://github.com/orgs/community/discussions/107446)

**Reply by Juli0787** [reply]
Hej Lili,
Many thanks for your understanding, and apologize me just coming back to the community today! Please find the errors below:
60 Error: GH_PAT secret is set but invalid or lacking appropriate privileges on the Juli0787/Match-Secrets repository. Verify that it is set correctly and try again.
99 Error: Process completed with exit code 2.
Ich highly appreciate your support.
Best,
Julia
